modules/cells/src/main/java/dmg/cells/services/CoreRoutingManager.java

```diff
@@ -329,7 +327,7 @@ public void messageArrived(CellMessage msg) {
                           .map(CellDomainInfo::getCellDomainName)
                           .forEach(domain -> domains.put(domain, new ArrayList<>()));
                     queueRoutes.asMap().forEach(
-                          (domain, cells) -> domains.put(domain, Lists.newArrayList(cells)));
+                          (domain, cells) -> domains.put(domain, new ArrayList<>(cells)));
```

Comment:
Maybe `List.copyOf()` is a better

Reply:
It might be, however List.copyOf() will return an immutable List. I will see if this is adequate in this case, if we have any modification then this is of course not adequate.